### DISCUSSION TITLE
New hardware and software variant

### DISCUSSION THREAD

**Discussion by 0unrouted0** [kachurovskiy/nanoels#200] [Ideas]
Hello,

i have started a new variant for myself with the intention of lower cost, hand soldering and newest Arduino esp32 core.
It's still in work and I'm far away (much weeks) from integrating it in my lathe and start some real work, but maybe the current ideas can help somebody else - or can be integrated in the original repo.

Current working in the unstable branch:
https://github.com/0unrouted0/nanoels/tree/unstable

The software under \h4_revised1\SW should be theoretically compatible with the original H4, but it's currently not completely tested. (BE CAREFUL on real lathe!)

The slightly modified software is working on ESP32 component 3.0.1 (Arduino) with an I2C display (if configured). Together with the keypad on an secondary I2C interface.

**Comment by kachurovskiy** [comment]
Hi @0unrouted0, awesome project, keep us updated on the progress. Will be happy to advertise this wider once you reach a stable version. Cheers!
